test/multiapi/Expected/AcceptanceTests/Multiapi/multiapi/v3/aio/operations_async/_operation_group_two_operations_async.py

```diff
@@ -73,9 +73,9 @@ async def test_four(
 
         # Construct and send request
         body_content_kwargs = {}  # type: Dict[str, Any]
-        if header_parameters['Content-Type'] in ['application/pdf', 'image/jpeg', 'image/png', 'image/tiff']:
+        if header_parameters['Content-Type'] or header_parameters['Content-Type'].split(";")[0] in ['application/pdf', 'image/jpeg', 'image/png', 'image/tiff']:
```

Comment:
I have the check for both content_type and content_type.split(";") since, for example, "text/plain; charset=utf-8" is a valid media type in swagger 2.0 and 3.0: https://swagger.io/docs/specification/media-types/